### PR TITLE
Addressing some post-release docs issues (DOC-7481)

### PR DESCRIPTION
**Pull request by paperclypse**
A grab bag of changes.

### CODE REVIEW

File: src/content/docs/codestream/start-here/install-codestream.mdx
Comment (bradleycamacho) [inline src/content/docs/codestream/start-here/install-codestream.mdx:17]: Is this the "preferred" way to install CodeStream? Just want to make sure the ordering of these makes sense.

File: src/content/docs/codestream/start-here/install-codestream.mdx
Comment (bradleycamacho) [inline src/content/docs/codestream/start-here/install-codestream.mdx:27]: Maybe link to https://code.visualstudio.com/docs/editor/extension-marketplace?


Could just add more confusion though.

**Review by bradleycamacho — DISMISSED** [review]

**Comment by bradleycamacho** [comment]
Made two small comments, but it looks good otherwise! Feel free to ignore and merge or implement and merge without a second review!

File: src/content/docs/codestream/start-here/install-codestream.mdx
Comment (paperclypse) [reply src/content/docs/codestream/start-here/install-codestream.mdx:17]: It's what makes the most sense to me and I haven't heard otherwise.

File: src/content/docs/codestream/start-here/install-codestream.mdx
Comment (paperclypse) [reply src/content/docs/codestream/start-here/install-codestream.mdx:27]: This is what the original CodeStream docs went with, so that's what I used.

**Pull request merged**
